index.html

```diff
@@ -0,0 +1,302 @@
+                <h1 class="headline-heading heading-text">Hey there. I'm Francis.
+                    <span id="color-this">I'm a software developer</span>
+                </h1>
+
+                <p class="headline-description description-text">I can help you build a product , 
+                    feature or website Look through 
+                    some of my work and experience! 
+                    If you like what you see and have 
+                    a project you need coded, 
+                    don't hestiate to contact me.</p>
+
+                <div id="headline-social-media">
+                    <div class="social-media">
+                        <section class="social-media-icon"><a href="http//:github.com/francis450"><img src="img/social-media-icons/github.svg"></img></a></section>
+                        <section class="social-media-icon"><a href="http//:linkedin/in/njoroge-francis"><img src="img/social-media-icons/linkedin.svg"></img></a></section>
+                        <section class="social-media-icon"><a href="http//:evangelist"><img src="img/social-media-icons/angelist.svg"></img></a></section>
+                        <section class="social-media-icon"><a href="http//:twitter.com/nkamandefrancis"><img src="img/social-media-icons/twitter.svg"></img></a></section>
+                        <section class="social-media-icon"><a href="http//:medium.com"><img src="img/social-media-icons/medium.svg"></img></a></section>
+                    </div>
+                </div>
+                
+                <div class="headline-to-bottom-icon">
+                    <a class="bottom-icon-container" href="#works"><img src="img/down.svg"></img></a>
+                </div>
+            </section>
+        </div>
+
+        <!-- Recent Works Section -->
+        <section id="works">
+            <div id="works-title">
+                <h2 class="works-title-text heading-text">My Recent Work</h2>
+            </div>
+
+            <div id="work-cards">
+                <section class="work-cards-row">
+                    <!-- card1 -->
+                    <div class="card">
+                        <section class="card-content">
+                            <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+                            
+                            <ul class="card-tags">
+                                <li class="card-tag ruby"><p class="card-tag-text">Ruby on rails</p></li>
+                                <li class="card-tag css"><p class="card-tag-text">css</p></li>
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
```

Comment:
- [ ] Kindly consider applying button interactions in `:hover` and `:active` states in CSS by using the colors guidelines in figma design and apply that in all the similar buttons in your website. the requirement is following:
    -  Implement the button interactions (enable, hover, etc.).
![qw](https://user-images.githubusercontent.com/96683843/219129631-417e0b10-fbe4-4ddb-8b3f-00022c640ff4.jpg)